index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="zxx">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+
+  <link rel="stylesheet" href="./Components/Pages/Styles/main.css">
+  <title>Game Ranking: Home page</title>
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-md navbar-light bgp-light">
+      <div class="container-fluid">
+        <a class="navbar-brand p-0 m-0" href="#"><img class="p-0 m-0 logo-container"
+            src="./Components/Pages/Assets/icon-game.gif" alt="icon"></a>
+        <button class="navbar-toggler me-auto" type="button" data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page"
+                href="./Components/Pages/About/Component/about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link disabled">News</a>
+            </li>
+            <li class="nav-item mx-5 nav-content">
+              <a class="nav-link">CC Campain</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="body-bg d-flex justify-content-center">
+      <div class="header">
+        <video class="d-flex justify-content-center" id="background-video" autoplay loop muted
+          poster="./Components/Pages/Assets/mkke-bg.png">
+          <source src="./Components/Pages/Assets/mkkefinal.mp4" type="video/mp4">
+        </video>
```

Comment:
- [ ] Although this is a really cool background, you should understand that adding such large files in your website may limit the accessibility to many of its potential users. The video you set as background for this section is really heavy (44MB), and for 95% of the users it would take more than a few seconds to load, resulting in a laggy and heavy website, not to mention it also makes cloning your repository really heavy and it won't look as good on mobile devices, making its loading pointless. I suggest you stick to the template and use the recommended background for this section, as it will improve the loading times and overall accessibility.